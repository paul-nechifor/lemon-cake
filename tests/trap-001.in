($trap 4)
($trap (+ 1 2))
($trap ($entrap 5))
($trap (+ 1 2 ($entrap 'a')))
($trap (if 1 ($entrap 'b')))
($trap (if 0 ($entrap 'b') (+ 4 4)))
($trap (+ 1 2 ($entrap (+ 8 8))))
