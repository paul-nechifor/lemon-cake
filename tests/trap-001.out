4
3
5
'a'
'b'
8
16
